atomic_reactor/schemas/workflow_data.json

```diff
@@ -33,11 +33,6 @@
     "koji_source_manifest": {"type": ["object", "null"]},
 
     "buildargs": {"type": "object"},
-    "built_image_inspect": {"type": ["object", "null"]},
-    "layer_sizes": {
-      "type": "array",
-      "items": {"$ref": "#/definitions/layer_size"}
```

Comment:
Can we also remove the layer_size definition now, or is it still used?

Reply:
removed